dashboard/craco.config.js

```diff
@@ -8,7 +8,11 @@ module.exports = {
   webpack: {
     configure: {
       plugins: [
-        new NodePolyfillPlugin(), // add the required polyfills (not included in webpack 5)
+        // add the required polyfills (not included in webpack 5)
+        new NodePolyfillPlugin({
+          // Allow using console.log
+          excludeAliases: ['console'],
+        }),
```

Comment:
Interesting... don't know how I haven't faced this problem in my local :S

Reply:
Try removing the modules folder, lock file and `yarn install` maybe.